packages/react-components/react-breadcrumb-preview/src/components/BreadcrumbLink/useBreadcrumbLink.ts

```diff
@@ -18,6 +18,9 @@ export const useBreadcrumbLink_unstable = (
 ): BreadcrumbLinkState => {
   const { appearance, iconPosition, size } = useBreadcrumbContext_unstable();
   const { current = false, disabled = false, icon, overflow = false, ...rest } = props;
+  const href = disabled ? undefined : props.href;
+  const as = props.as || (href ? 'a' : 'button');
+  const type = as === 'button' ? 'button' : undefined;
```

Comment:
It's the same behavior as in FluentLink component. It needs to pass disabled state

Reply:
Not sure if this was discussed previously, but if `BreadcrumbLink` and `Link` have props in common and they share the same behavior, it might make sense to reuse the logic of the `useLink_unstable`. With that, props like `href`, `as` and `type` would receive the same "treatment" and be reflected equally in both components. What do you think?

Reply:
Thank you for the help with this :)